### DISCUSSION TITLE
how to update ytmusic dependency(s)?

### DISCUSSION THREAD

**Discussion by EquinoxRevolt** [pear-devs/pear-desktop#2699] [Q&A]
not sure if this is important enough to write within an issue so im writing here, in the few posts on the unexpected token error with the downloader plugin they are redirected to #2681, 
it'd be awesome if i could get a push in the right direction or any kind of answer on how to update this dependency and make the downloader plugin work, as im only used to be babyfed installers that do everything for me 🤥
thanks in advance

**Comment by Pokobonzi** [comment]
I have the same problem with the downloader giving the error, so how to update it.
![Screenshot_2](https://github.com/user-attachments/assets/a219a669-5d2b-4445-ab23-9a321b0df18a)
